standalone/runtime/src/lib.rs

```diff
@@ -594,6 +596,8 @@ impl pallet_dkg_metadata::Config for Runtime {
 	type OffChainAuthId = dkg_runtime_primitives::offchain::crypto::OffchainAuthId;
 	type NextSessionRotation = pallet_dkg_metadata::DKGPeriodicSessions<Period, Offset, Runtime>;
 	type RefreshDelay = RefreshDelay;
+	type UnsignedPriority = UnsignedPriority;
+	type UnsignedInterval = UnsignedInterval;
```

Comment:
We also need to add the `ValidateUnsigned` check to SignedExtra, like https://github.com/paritytech/polkadot/blob/e3f67317030ce6358596780370c8d221f3a65191/runtime/polkadot/src/lib.rs#L1540

Reply:
I do believe there are no need for this check, since we do validate the unsigned transaction anyway in the pallet, and reject it before it hits the actual extrinsic, actually we do not even put it in the txpool.